### ISSUE TITLE
Spotify namespace support

### ISSUE THREAD

**Issue by mpgirro** [mpgirro/wien#29]
Specification file: https://podcasters.spotify.com/terms/Spotify_Podcast_Delivery_Specification_v1.6.pdf

**Comment by rock3r** [comment]
FYI latest version is 1.8, somewhat confusingly hosted on Google Drive https://drive.google.com/file/d/1KDY1zbRc6J2tkNvhniagor_qcH-pp2T0/view

Via https://support.spotifyforpodcasters.com/hc/en-us/articles/360044440991-Podcast-specification-doc

I have also added an item to the readme to indicate we intend to support it.

**Comment by mpgirro** [comment]
Ah, good that you've spotted that! My Google search still returns the 1.6 document 😕

**Comment by rock3r** [comment]
Yeah I have had to dig a bit to find it, but have confirmed with Spotify support it's the latest. Also have reported at least a major error in that spec, while we're at it :) so I expect a 1.9 version to come out soon. Later today I'll put here a list of tags to support, there's just a handful anyway.

**Comment by rock3r** [comment]
### Namespace

 Prefix | URI
 --- | ---
 `spotify` | `https://www.spotify.com/ns/rss`

### Tags

<table>
  <thead>
    <tr>
      <th>Scope</th>
      <th>Tag</th>
      <th>Count</th>
      <th>Notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td rowspan=2><code>&lt;channel&gt;</code></td>
      <td><code>&lt;spotify:limit recentCount="10" /&gt;</code></td>
      <td>0 or more</td>
      <td>
        <ul>
          <li>Makes no sense to have more than 1, so I'll allow only 1</li>
          <li><code>recentCount</code> is how many recent episodes to display in Spotify, a strictly positive integer</li>
        </ul>
      </td>
    <tr>
      <td><code>&lt;spotify:countryOfOrigin&gt;...&lt;/spotify:countryOfOrigin&gt;</code></td>
      <td>0 or 1</td>
      <td>
        <ul>
          <li>Territories where podcast is relevant</li>
          <li>Value is list of space-separated ISO 3166 country codes. E.g., <code>us gb es</code>code></li>
          <li>When missing, a podcast is "global"</li>
        </ul>
      </td>
    </tr>
</table>

**Comment by rock3r** [comment]
According to https://github.com/mpgirro/stalla/issues/81#issuecomment-824577901, these attributes are  almost never used in the wild, so the priority to support them is very low.
